src/auditlog/models.py

```diff
@@ -94,6 +94,9 @@ def get_for_objects(self, queryset):
 
         if isinstance(primary_keys[0], integer_types):
             return self.filter(content_type=content_type).filter(Q(object_id__in=primary_keys)).distinct()
+        elif isinstance(queryset.model._meta.pk, models.UUIDField):
```

Comment:
@grantmcconnaughey agreed that is the advantage of using the `Meta`, I understand @jjkester trying to keep the code consistent to improve readability, but the current implementation does cause issues in some cases. I don't know how you replace the first check to see if it is of type (int/long) though, you can do `isinstance(queryset.model._meta.pk, models.AutoField)` but you would have to handle the other cases as well.